### ISSUE TITLE
Use new font faces

### ISSUE THREAD

**Issue by DenisHdz** [bbc/simorgh#1650]
Blocked on bbc/psammead#479

**Is your feature request related to a problem? Please describe.**
Use new font faces, switching ones used based on service.

**Describe the solution you'd like**

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

**Additional context**

**Comment by dr3** [comment]
Closing as not needed for V1, as theres no description, no harm in remaking this when needed
